client-src/elements/chromedash-form-field.ts

```diff
@@ -179,7 +187,7 @@ export class ChromedashFormField extends LitElement {
     }
     // There should only be one now.
     const formFieldElement = formFieldElements[0];
-
+    let checkResult;
```

Comment:
That's interesting: if `checkResult` didn't exist here, then the below line would either read `window.checkResult` (which I don't think is created anywhere) or throw. @dlaliberte, what's going on here?

Instead of creating a local variable that'll always be `undefined`, this PR should fix or delete the logic.

Reply:
will fix when @dlaliberte throw some hints here :))

Reply:
Ok, what's going on is that this function runs before the `<sl-input>` element finishes upgrading, so `formFieldElement.input` is always false. As far as I can see, that means `setupSemanticCheck()` is always a no-op, so we should just delete it instead of fixing up its types. Any objections?

Reply:
You appear to be correct, Jeffrey.  I don't recall what happened regarding this checkResult value.  But I also don't recall leaving it as a noop.  If the semantic checks are still working, and assuming tests are sufficient, then this function can go.  I do recall having difficulty synchronizing the initial semantic check call when first loading a page that has pre-existing errors in the data.  So be sure to check for that situation.